CONTRIBUTING.md

```diff
@@ -235,3 +238,12 @@ directory. As an expedient to make the third_party code work well with our build
 code in third_party should be given a stub pubspec.yaml file so that you can
 reference the resources from the packages directory from
 `packages/devtools_app/web/index.html`
+
+## Enabling and activating DCM
+
+1. Install the executable for your target platform. You can refer to [this guide](https://dcm.dev/docs/teams/getting-started/#installation).
```

Comment:
Before the list of steps, let's add:

```
> Note: Enabling and activating DCM is optional. When you open a PR, the CI bots will show you any DCM warnings introduced by your change which should be fixed before submitting.

**[Contributors who work at Google]** You can use the Google-purchased license key to activate DCM. See [go/dash-dcm-keys](http://goto.google.com/dash-dcm-keys)

**[All other contributors]** Please follow instructions at https://dcm.dev/pricing/. You can either use the free tier of DCM, or purchase a team license.
```

This way the instructions are applicable to everyone.

Reply:
Also instead of  `[go/dart-devtools-dcm-key](http://goto.google.com/dart-devtools-dcm-key)`, let's use the following url:  [go/dash-devexp-dcm-keys](http://goto.google.com/dash-devexp-dcm-keys)

Reply:
> You can either use the free tier of DCM

So this might be confusing (see https://github.com/flutter/devtools/pull/5629#discussion_r1170272026), are you sure you want to mention it as an option even though they (free and paid) are not fully compatible?

Reply:
Ah got it. Hmm maybe the wording should be:

**[All other contributors]** Please follow instructions at https://dcm.dev/pricing/. You can either use the free tier of DCM, or purchase a team license. Note that the free tier doesn't support all the rules of the paid tier, so you will also need to consult the output of the Dart Code Metrics workflow on Github when you open your PR.

Reply:
Done